scary_game.py

```diff
@@ -0,0 +1,293 @@
+
+    # This is where you can use the Fire lamp
+    def use_fire_lamp(self, backpack, item_path):
+        if matches in backpack:
+            print(f"The item is on {item_path}. \n")
+            backpack.remove(matches)
+        else:
+            print("You do not have a fire lamp\n")
+
+    # This is where you can use the pistol.
+    def use_pistol(self, backpack):
+        if deagle in backpack and bullets in backpack:
+            print("You need to pick the path with the demon and you will kill him and get your daughter back\n")
+            backpack.remove("Bullets")
+        else:
+            print("You do not have all the parts")
+
+    def create_gun(self, backpack):
+        if pistol in backpack and magazine in backpack:
+            # This are the items being removed from the backpack
+            backpack.remove(magazine)
+            backpack.remove(pistol)
+
+            # The user gains a item since he had all the required parts
+            backpack.append(deagle)
+            backpack.sort()
+
+
+# This is the evil demon object, and it will be sent to the game_engine
+class Evil_demon(object):
+    """
+    This class is the demon class and really you just die if you encounter the demon.
+    """
+
+    # If  the user picks this path, the user dies
+    def death(self):
+        print("The demon ate you for dinner and let your child go back home parentless.")
+        exit(1)
+
+    def kill_demon(self):
+        print("You have killed the demon")
+        print("Daughter: DADDY you saved me, I knew you would come rescue me!!")
+        exit(0)
+
+    def missed_demon(self):
+        print("You shot but missed the demon. You will have to find another bullet and try again.")
+
+
+# This is the item object, and will be sent to the game engine as well.
+class Item(object):
+    """
+    This is the item class where the item list and the user backpack is intialized and if
+    you walk the item path then you collect that item. Once the item has been used then it gets removed.
+    """
+
+    # These are the variables that are in this class. and will only be in this class.
+    def __init__(self, items_list, users_backpack):
+        self.items_list = items_list
+        self.users_backpack = users_backpack
+
+    # If the user picks this path, the user collects the item and will be sent to his backpack
+    def collect_item(self):
+        prize_item = random.choice(self.items_list)
+        print(f"You have survived and you have found => {prize_item}")
+        self.users_backpack.append(str(prize_item))
+        self.users_backpack.sort()
+
+        # The user should not collect the same gun parts twice. It is a one and done thing
+        if prize_item in [magazine, pistol]:
+            self.items_list.remove(prize_item)
+
+
+
+# instanciating all the classes to objects
+
+def main():
+    # instanciating all the classes to objects
+    Anunnaki = Evil_demon()
+    Ali = Father()
+
+    This_the_backpack = [battery]
```

Comment:
`This_the_backpack` should not have capital letters. Also remember variable names should be 1-2 words long